### ISSUE TITLE
any suggestion on converting shufflenet structure in TF to pytorch 

### ISSUE THREAD

**Issue by MrWanter** [Microsoft/MMdnn#140]
Hi! thanks for your work! I'm currently facing an issue that I want to convert a `shufflenet` pretrained model in TF to pytorch format, as the network contains` group convolution` and `depth convolution` as well as `concat op`, also it's in `NCHW` format, could you give some suggestion or if it is currently possible in MMdnn? Thanks !

**Comment by namizzz** [comment]
Hi, @MrWanter could you provide the TF model files (ckpt ckpt.data ckpt.meta or .pb)?
